packages/spright-components/package.json

```diff
@@ -0,0 +1,129 @@
+    "pack": "npm pack",
+    "invoke-publish": "npm publish",
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build -o dist/storybook --webpack-stats-json",
+    "storybook-open-webkit": "playwright wk http://localhost:6006",
+    "build-components": "tsc -p ./tsconfig.json",
+    "build-components:watch": "tsc -p ./tsconfig.json -w",
+    "bundle-components": "rollup --bundleConfigAsCjs --config",
+    "tdd": "npm run build-components && npm run test-chrome",
+    "tdd:watch": "npm run build-components:watch & npm run test-chrome:watch",
+    "tdd-firefox": "npm run build-components && npm run test-firefox",
+    "tdd-firefox:watch": "npm run build-components:watch & npm run test-firefox:watch",
+    "tdd-webkit": "npm run build-components && npm run test-webkit",
+    "tdd-webkit:watch": "npm run build-components:watch & npm run test-webkit:watch",
+    "test-chrome:debugger": "karma start karma.conf.js --browsers=ChromeDebugging --skip-tags SkipChrome",
+    "test-chrome:verbose": "karma start karma.conf.verbose.js --browsers=ChromeHeadlessOpt --single-run --skip-tags SkipChrome",
+    "test-chrome:watch": "karma start karma.conf.js --browsers=ChromeHeadlessOpt --skip-tags SkipChrome --watch-extensions js",
+    "test-chrome": "karma start karma.conf.js --browsers=ChromeHeadlessOpt --single-run --skip-tags SkipChrome",
+    "test-firefox:debugger": "karma start karma.conf.js --browsers=FirefoxDebugging --skip-tags SkipFirefox",
+    "test-firefox:verbose": "karma start karma.conf.verbose.js --browsers=FirefoxHeadless --single-run --skip-tags SkipFirefox",
+    "test-firefox:watch": "karma start karma.conf.js --browsers=FirefoxHeadless --skip-tags SkipFirefox --watch-extensions js",
+    "test-firefox": "karma start karma.conf.js --browsers=FirefoxHeadless --single-run --skip-tags SkipFirefox",
+    "test-webkit:debugger": "karma start karma.conf.js --browsers=WebkitDebugging --skip-tags SkipWebkit",
+    "test-webkit:verbose": "karma start karma.conf.verbose.js --browsers=WebkitHeadless --single-run --skip-tags SkipWebkit",
+    "test-webkit:watch": "karma start karma.conf.js --browsers=WebkitHeadless --skip-tags SkipWebkit --watch-extensions js",
+    "test-webkit": "karma start karma.conf.js --browsers=WebkitHeadless --single-run --skip-tags SkipWebkit",
+    "test": "npm run test-chrome:verbose && npm run test-firefox:verbose"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ni/nimble.git"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "author": {
+    "name": "National Instruments"
+  },
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/ni/nimble/issues"
+  },
+  "homepage": "https://github.com/ni/nimble#readme",
+  "dependencies": {
+    "@microsoft/fast-colors": "^5.3.1",
+    "@microsoft/fast-element": "^1.12.0",
+    "@microsoft/fast-foundation": "^2.49.6",
+    "@microsoft/fast-web-utilities": "^6.0.0",
+    "@ni/nimble-components": "*",
```

Comment:
Thanks, I had been thinking of `"*"` as "latest", since it allows the newest version, but of course that's not what it means. Switched it back.

Reply:
"latest" does exist but we should use semantic version ranges to specify the version we actually test against. i.e. latest changes day-to-day and doesn't care about major versions, etc. If you want to make sure the mental model for how semver works is aligned we should discuss offline as you're setting up more packages.

Reply:
This came up in the wild for a third party package using wildcard specifiers for dependencies as an example: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69355#issuecomment-2088994473

Even though the linkify-it package did a major version release to specify a breaking change users of markdown-it (directly or indirectly) were broken by the major change because markdown-it used a wildcard specifier.